app/radiosonde_auto_rx_listener.py

```diff
@@ -21,10 +21,97 @@ def __init__(self):
 
         logger.info("AsyncRadiosondeAutoRxListener initialized.")
 
-
     async def start(self):
+        logger.info("Starting AsyncRadiosondeAutoRxListener...")
+        if self._settings.fetch_data:
+            logger.info("Fetching data from online sites.")
+            await self._listen_web()
+        else:
+            logger.info("Fetching data from local udp.")
+            await self._listen_udp()
+
+    async def _listen_web(self):
+        """Uses async requests to fetch data from online sources."""
+        url = f"https://s1.radiosondy.info/export/export_map.php?live_map=1&_={int(datetime.now().timestamp() * 1000)}"
```

Comment:
I think the urls must be updated ad each iteration with the new timestamp, or not?